tdms/include/globals.h

```diff
@@ -53,6 +53,7 @@ enum AxialDirection
 enum ModeOfRun    { Pass1 , Pass2 };
 enum RCSType      { parallel , perpendicular };
 enum SolverMethod { PseudoSpectral, FiniteDifference };
+enum PreferredInterpolationMethods { BandLimited, Cubic };

```

Comment:
```suggestion
enum PreferredInterpolationMethod { BandLimited, Cubic };

```
Why not singular? Do I miss something?